### PR TITLE
Minor Changes to Surveys

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Performs minor changes requested by the user to all surveys. These are needed primarily for the Fall survey at this point, but have been applied to all surveys for uniformity (and because the clients have confirmed that they do want these changes on all surveys).

Connects #532 

### Demo

Negative values are not allowed in numeric fields anymore:

Text changes to the Survey tab:

image

"Sell nucs/packages" in the User Survey:

image

Notes field on Monthly surveys:

image

Notes field on November surveys:

image

Notes field on April surveys:

image

## Testing Instructions

* Check out this branch
* Run all migrations `$ ./scripts/manage.sh migrate`
* Run beekeepers `$ ./scripts/beekeepers.sh start`
* Sign up as a new user to see the User Survey
  - [x] Ensure you see "Sell nucs/packages" under "Do you receive income from any of the following?"
* Go to the Surveys tab
  - [x] Ensure you see the new language
* Open different kinds of Apiary surveys to verify the Notes field
  - [x] Ensure the field is optional, and you can leave it blank
  - [x] Ensure it is saved correctly in November Surveys
  - [x] Ensure it is saved correctly in April Surveys
  - [x] Ensure it is saved to the right colony in Monthly Surveys
* On the different kinds of surveys, try entering negative values in the numeric fields
  - [x] Ensure direct input of negative values is not allowed
  - [x] Ensure negative values cannot be input using <kbd>up</kbd> and <kbd>down</kbd> keys

### CODE REVIEW

**Comment by rajadain** [comment]
> It is mysterious because I can't find where notes is `null` or `undefined`.

Fixed in e544d2c by defaulting to `''` in case of `null` or `undefined`.

File: src/icp/apps/beekeepers/js/src/components/SurveyView.jsx [hunk 1]
```diff
@@ -74,17 +74,17 @@ const SurveyView = ({ apiaries, isProUser }) => {
                     }
                 </div>
                 <div className="survey__body--section">
-                    <h2 className="survey__title">Up to date</h2>
+                    <h2 className="survey__title">Apiaries with all surveys completed:</h2>
```
[the remark this answers is not in the document]
Comment (rajadain) [reply src/icp/apps/beekeepers/js/src/components/SurveyView.jsx]: Added more changes to make it look like the client's mock-up in e544d2c. Here's a screenshot: https://prnt.sc/pmrig8 (couldn't upload to GitHub for some reason).

**Comment by rajadain** [comment]
Ready for another look.

**Comment by rajadain** [comment]
Thanks for reviewing!

**Comment by dboyer** [comment]
Does this task also include:

•	Add “N/A – no colonies lost” option in Spring survey for the “What do you think was the most likely cause of colony loss?” question

**Comment by rajadain** [comment]
No, this did not. May have missed that between the Google Doc and the GitHub Issue.

If there's any more additions, we can collect them in a new issue that can be brought on as a ➕ card.

**Pull request merged**